etc/ReplayOfflineConfiguration.py

```diff
@@ -35,8 +37,19 @@
 
 # Settings up sites
 processingSite = "T0_CH_CERN"
+storageSite = "T0_CH_CERN_Disk"
 streamerPNN = "T0_CH_CERN_Disk"
 
+addSiteConfig(tier0Config, "T0_CH_CERN_Disk",
+                siteLocalConfig="/cvmfs/cms.cern.ch/SITECONF/T0_CH_CERN/JobConfig/site-local-config.xml",
```

Comment:
Due that this siteLocalConfig and overrideCatalog have a default value within the function addSiteConfig, Is it mandatory to use it again here in the OfflineConfig file??

Reply:
Yes, it is not mandatory. But I think is worth adding it explicitly, so it is clear to everyone what values we are using. Like when we add a default Repack config or default dataset config.